regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -105,6 +112,26 @@ var CommentReviewView = Backbone.View.extend({
     });
   },
 
+  initToggles: function() {
+    function toggle($elm) {
+      var $target = $('#' + $elm.data('toggle'));
```

Comment:
Minor -- do we think we should allow this to look at the full doc rather than just ? I can see an argument either way.